### PR TITLE
Feature/nm 56

### PR DESCRIPTION
**Pull request by tomasmihaly**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -25,6 +25,8 @@ app.get('/github/teamstats', (req, res, next) => github.getGithubDataForTeam(req
 
 app.get('/github/contribution', (req, res, next) => contribution.getContribution(req, res, next));
 
+app.get('/github/teamsScores', (req, res, next) => impact.getTeamsScores(req, res, next));
```
Comment (tjalc) [inline app.js]: we should not use /github prefix

File: controllers/impact.controller.js [hunk 2]
```diff
@@ -40,7 +40,28 @@ function getTeamImpact(req, res, next) {
         });
 }
 
+function getTeamsScores(req, res, next) {
+
+    let startDate = req.query.startDate;
+    let endDate = req.query.endDate;
+
+    if (!startDate || !endDate) {
+        throw new Error(`Missing one of required parameters: startDate = ${startDate}, endDate = ${endDate}`)
```
Comment (tjalc) [inline controllers/impact.controller.js]: are startDate and endDate required params? did we agreed with Middleware team on that?

Comment (tomasmihaly) [reply controllers/impact.controller.js]: I think they should be required as requesting all the commits in a repository can be very expensive call

**Comment by tomasmihaly** [comment]
commited fix for the method calls

File: controllers/impact.controller.js [hunk 3]
```diff
@@ -40,7 +40,28 @@ function getTeamImpact(req, res, next) {
         });
 }
 
+function getTeamsScores(req, res, next) {
```
Comment (ajancus) [inline controllers/impact.controller.js]: Isnt better to move this metric to own controller and service? Then impact service will be more cleaner.

Comment (tomasmihaly) [reply controllers/impact.controller.js]: refactored, committed, please let know if its ok

File: controllers/impact.controller.js (hunk 2 above)
Comment (tomasmihaly) [reply controllers/impact.controller.js]: are we ok with this?

**Comment by ajancus** [comment]
Reslove conflicts before merging.

**Pull request merged**